projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+
+***Environment variable***
+: `XRAY_PROJECTKEY`
+
+***Type***
+: `string`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "projectKey": "PRJ"
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_PROJECTKEY="PRJ"
+        ```
+
+<hr/>
+
+## `resultsUpload`
+
+These settings determine the plugin's behaviour regarding the results upload.
+
+### `assignee`
+
+The Jira user to assign created test executions to.
+The following values should be provided here:
+
+- :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } The account ID, usually of the form `61f8f589e688d6007068a792`.
+  You can retrieve account IDs by exporting an issue as XML where the user is visible (reporter, assignee, ...). The XML element containing the account ID will then look something like this:
+  ```xml
+  <assignee accountid="61f8f589e688d6007068a792">John Miller</assignee>
+  ```
+- :fontawesome-solid-server:{ title="Xray server" .xray-icon } The username used for login, for example `miller_j`.
+  You can retrieve usernames of other users by exporting an issue as XML where the user is visible (reporter, assignee, ...). The XML element will then look something like this:
+  ```xml
+  <assignee>miller_j</assignee>
+  ```
+
+!!! note
+    If the configured assignee is `#!json null`, the test execution issues will not be assigned to anyone.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_ASSIGNEE`
+
+***Type***
+: `string`
+
+***Default***
+: `#!json null`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "resultsUpload": {
+            "assignee": "61f8f589e688d6007068a792"
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_ASSIGNEE="61f8f589e688d6007068a792"
+        ```
+
+<hr/>
+
+### `customStatus`
+
+These status settings map QTAF test (execution) statuses to corresponding Jira issue or Xray test and step statuses.
+
+<hr/>
+
+#### `step`
+
+These settings allow mapping QTAF test _steps_ to Xray _step_ statuses.
+
+!!! tip
+    If you have custom step statuses set up in Xray, you should provide their names here.
+
+<hr/>
+
+##### `failed`
+
+The Xray status of failed steps.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_STEP_FAILED`
+
+***Type***
+: `string`
+
+***Default***
+: :fontawesome-solid-server:{ title="Xray server" .xray-icon } `#!json "FAIL"`
+: :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } `#!json "FAILED"`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "customStatus": {
+              "step": {
+                "failed": "ERROR"
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_STEP_FAILED="ERROR"
```

Comment:
It should be CUSTOMSTATUS and not CUSTOM_STATUS
This error occurs multiple times in this document.

Reply:
Fixed in 738882cd3386314984a1eb1166c08b46dd6cf2fb.